File: game/src/main/java/Card.java
public class Card {
    private String sign;
    private boolean reversed;

    public Card (String sign) {
        this.sign = sign;
        this.reversed = false;
    }


    public String getSign() {
        return sign;
    }

    public void setSign(String sign) {
        this.sign = sign;
    }

    public boolean isReversed() {
        return reversed;
    }

    public void setReversed(boolean reversed) {
        this.reversed = reversed;
    }
}

File: Card.cpp
#include <string>
using namespace std;

class Card {
    private:
        string sign;
        bool reversed;
    public:
		Card (string sign) {
			this->sign = sign;
			this->reversed = false;
		}
		
		string getSign() {
			return sign;
		}

		void setSign(string sign) {
			this->sign = sign;
		}

		bool isReversed() {
			return reversed;
		}

		void reverse(bool reversed) {
			this->reversed = reversed;
		}
};
